topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -988,55 +1415,207 @@ goseq needs 2 files as inputs:
 >    - *"Delimited by"*: `Tab`
 >    - *"To"*: `Upper case`
 >
->    We just generated the first input for goseq. We need as second input for goseq, the gene lengths. We can use there the gene length generated by **featureCounts** {% icon tool %} and reformat it a bit.
+> 4. Rename the output to `Gene IDs and differentially expression`
+>
+>    We just generated the first input for **goseq**. As second input for **goseq** we need the gene lengths. We can use here the gene lengths generated by **featureCounts** {% icon tool %} and format the gene IDs.
+>
+> 5. Drag and drop of the gene length file generated by **featureCounts** {% icon tool %} into this history using the {% icon galaxy-columns %} **View all histories**
```

Comment:
Here, it is not clear which files. In the first tutorial history there are 2 feature counts lengths files. One from GSM461180 and one from GSM461177. Shall I take both (I did now to see further in the tutorial)

Reply:
Only one file is needed: I changed to "Drag and drop one of the gene length file"